mm2src/coins/utxo/slp.rs

```diff
@@ -2112,7 +2101,7 @@ mod slp_tests {
         };
         let validity_err = block_on(fusd.validate_htlc(input)).unwrap_err();
         match validity_err.into_inner() {
-            ValidateHtlcError::InvalidSlpUtxo(e) => println!("{:?}", e),
+            ValidatePaymentError::InvalidInput(e) => println!("{:?}", e),
```

Comment:
I'm afraid that here we need to expect `WrongPaymentTx`, not `InvalidInput`. Please check if everything is correct.